### DISCUSSION TITLE
Adding extra logic to the CreateOrder class

### DISCUSSION THREAD

**Discussion by DavidKimmer** [lunarphp/lunar#1191] [Q&A]
I need to add some extra logic when creating an order but the CreateOrder class doesn't utilise the HasModelExtending trait. How can I extend the CreateOrder class to implement the extra logic required? Thanks!

**Comment by ryanmitchell** [comment]
In 0.4+ CreateOrder has become a pipeline allowing you to extend it with whatever logic you want.
